test/Microsoft.Extensions.Localization.Tests/ResourceManagerStringLocalizerFactoryTest.cs

```diff
@@ -1,17 +1,79 @@
 ﻿// Copyright (c) .NET Foundation. All rights reserved. 
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
+using System.IO;
 using System.Reflection;
 using Microsoft.AspNetCore.Hosting;
 using Microsoft.Extensions.Options;
 using Moq;
 using Xunit;
 
-namespace Microsoft.Extensions.Localization.Test
+namespace Microsoft.Extensions.Localization.Tests
```

Comment:
Right, we're breaking the [guidelines](https://github.com/aspnet/Home/wiki/Engineering-guidelines#unit-test-class-naming) uniformly:
> Test class names end with Test and live in the same namespace as the class being tested.

But you're also right that this is a separate issue from this PR.